src/planwise/component/scenarios.clj

```diff
@@ -54,7 +54,15 @@
   (when (some? raster)
     (io/delete-file (io/file (str "data/" raster ".tif")) :silent)
     (io/delete-file (io/file (str "data/" raster ".map.tif")) :silent)
-    (io/delete-file (io/file (str "data/" raster ".coverage.tif")) :silent)))
+    (io/delete-file (io/file (str "data/" raster ".coverage.tif")) :silent)
+    (io/delete-file (io/file (str "data/" raster ".base-demand.tif")) :silent)
+
+    (let [old-provider-ids (set (keys (:new-providers-geom scenario)))
```

Comment:
This is obsolete code which was removed in c9f0c43c6d7861558e7b80946e6f985ec37f6ce6

Reply:
My bad. This is a merge error